xbmc/games/dialogs/osd/DialogGameVideoFilter.h

```diff
@@ -34,14 +34,16 @@ class CDialogGameVideoFilter : public CDialogGameVideoSelect
   unsigned int GetFocusedItem() const override;
   void PostExit() override;
   bool OnClickAction() override;
+  void RefreshList() override;
 
 private:
   void InitScalingMethods();
   void InitVideoFilters();
+  void InitGetMoreButton();
+  void OnGetMore();
+  void OnGetMoreComplete();
 
-  static void GetProperties(const CFileItem& item,
-                            std::string& videoFilter,
-                            std::string& description);
+  static void GetProperties(const CFileItem& item, std::string& videoFilter);
```

Comment:
Encapsulate as much as you can. Is there a reason for not putting this as a helper function into an unnamed namespace in the cpp file?

Reply:
Good point, will move.